Milestone3/frontend/src/App.js

```diff
@@ -0,0 +1,35 @@
+import logo from './logo.svg';
+import './App.css';
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+import Home from './home';
+import Template from './template';
+import recipes from './recipeData';
+import AboutMe from './aboutme';
+
+function App() {
+    return (
+        <Router>
+            <Routes>
```

Comment:
so looking through everything, the only requirement you are missing is the navbar part. This should include links back to your home page and about me, so users can navigate without using the browser arrows. it could also include the "Michelle's cookbook" header you have on each page. 
This is a pretty small component that you would just put above this routes tag so it displays on every page, no matter where you navigate to.

Reply:
got the navbar component now, and finished up milestone four. let me know what you think!